Tests/iaas/flavor-naming/flavor-names-openstack.py

```diff
@@ -82,7 +87,18 @@ def main(argv):
         print(f"Extra arguments {str(args)}", file=sys.stderr)
         usage(1)
 
-    scsMandatory = fnmck.readmandflavors(scsMandFile)
+    scsMandatory, scsRecommended = fnmck.readmandflavors(scsMandFile)
+    if not v3mode:
+        scsMandatory = [*scsMandatory, *scsRecommended]
+        scsRecommended = []
+        # HACK: Drop SSD flavors from list
+        scsV2Mand = []
+        for flv in scsMandatory:
+            ret = fnmck.parsename(flv)
+            disk = ret[1]
+            if not disk or "disktype" not in disk.__dict__ or disk.disktype != "s":
```

Comment:
This is `__dict__` check is quite hard to swallow for my taste. Could we instead use `not disk.parsed`? Would that do the trick as well?

Reply:
```suggestion
            if not disk or not disk.parsed or disk.disktype != "s":
```

Reply:
On second thought, I think the list `scsV2Mand` should be constructed via list comprehension, and then the variable could be dropped for good. Something like so:

```python
def is_ssd_flavor(name):
    disk = fnmck.parsename(flv)[1]
    return disk and disk.parsed and disk.disktype == "s"

scsMandatory = [name for name in scsMandatory if not is_ssd_flavor(name)]
```

Reply:
List comprehensions are definitely more readable, thanks.
Implemented in https://github.com/SovereignCloudStack/standards/pull/294/commits/642aa69165fc2712e0e807efc2e7d8f770454027